### PR TITLE
Create PULL_REQUEST_TEMPLATE.md & ISSUES_TEMPLATE.md

### PR DESCRIPTION
**Pull request by mgkwill**
Pull request & issue templates to standardize how people open pull requests and align with lava-nc/lava.

### CODE REVIEW

File: ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+<!-- Insert one sentence issue objective here, can be copied to PR. -->
+Objective of issue:
+
+<!-- Lava Version bug found in or Lava Version feature is targeting, remove unused versions from Lava Version below-->
+
+**Lava version:**
+- [ ] **0.3.0** (feature release)
+- [ ] **0.2.1** (bug fixes)
+- [x] **0.2.0** (current version)
+- [ ] **0.1.2**
```
Comment (mathisrichter) [inline ISSUE_TEMPLATE.md:22]: Shouldn't there be an additional question about the version of lava-dnf that this issue is targeting?

File: ISSUE_TEMPLATE.md [hunk 2]
```diff
@@ -0,0 +1,68 @@
+<!-- 
+
+- Before submitting an issue please refer to https://lava-nc.org/developer_guide.html#how-to-contribute-to-lava
+
+- Please make sure you are posting an issue pertaining to the github.com/lava-nc/lava-dnf, for issues with lava please file in lava-nc/lava repository, for example github.com/lava-nc/lava/issues
```
Comment (mathisrichter) [inline ISSUE_TEMPLATE.md:5]: "for issues with lava" -> "for issues with Lava"

Comment (mathisrichter) [inline ISSUE_TEMPLATE.md:5]: "pertaining to the" -> "pertaining to" ?

**Review by mathisrichter — CHANGES_REQUESTED** [review]
Some minor changes, all else looks good

File: ISSUE_TEMPLATE.md (hunk 1 above)
Comment (mgkwill) [reply ISSUE_TEMPLATE.md]: Yes.

**Pull request merged**